gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/main/java/io/gravitee/rest/api/management/v2/rest/mapper/ApplicationMapper.java

```diff
@@ -36,5 +36,6 @@ public interface ApplicationMapper {
     List<BaseApplication> mapToBaseApplicationList(Collection<ApplicationListItem> applications);
 
     @Mapping(target = "picture", source = "pictureUrl")
+    @Mapping(target = "disableMembershipNotifications", source = "notifyMembers")
```

Comment:
We need to negate notifyMembers to feed disableMembershipNotifications 

something like 

` @Mapping(target = "disableMembershipNotifications", expression = "java(!spec.isNotifyMembers())")`